sync-todo/v2/client/flutter/lib/components/todo_list.dart

```diff
@@ -92,4 +116,14 @@ class _TodoListState extends State<TodoList> {
       ],
     );
   }
+
+  launchURL({required String url}) async {
```

Comment:
```suggestion
  _launchURL({required String url}) async {
```

make private method.

also personally, i would make this a private func that's scoped to file level so it's more testable (even if you aren't actually writing tests for it)

Reply:
Oops. Yeah, that's a great point. Thank you!